apps/web/src/pages/settings/tabs/ApiKeysCard.tsx

```diff
@@ -27,7 +27,7 @@ export const ApiKeysCard = () => {
             readOnly
             type={'password'}
             rightSection={
-              <Tooltip data-test-id={'Tooltip'} label={clipboardApiKey.copied ? 'Copied!' : 'Copy Key'}>
+              <Tooltip data-test-id={'api-key-Tooltip'} label={clipboardApiKey.copied ? 'Copied!' : 'Copy Key'}>
```

Comment:
```suggestion
              <Tooltip data-test-id={'api-key-tooltip'} label={clipboardApiKey.copied ? 'Copied!' : 'Copy Key'}>
```
Seems we are mostly following a pattern of kebab-case data-test-id all in lowercase. Would you mind to update?

Reply:
Sorry, I was careless about it. 
It's fixed at 4a2068b1912b7c22ef50872864eb3d8cbc9d17be